### ISSUE TITLE
PCA after feature extraction

### ISSUE THREAD

**Issue by SomeSnm** [blue-yonder/tsfresh#42]
Hello. 

Am I understanding it correctly, that it is recommended to perform PCA on the extracted features to achieve better accuracy in classification? 

I saw it in the paper: https://arxiv.org/pdf/1610.07717v1.pdf

**Comment by MaxBenChrist** [comment]
In our simulations, this combination proved to be very competitive. You can use the [sklearn PCA implementation](http://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html)
